lib/module_importer.js

```diff
@@ -56,6 +65,7 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
     var relativePath = fragments.slice(1).join("/");
     var pkgRootDir = isRealFile ? importUtils.packageRootDir(path.dirname(prev)) : root;
     var jsFile = importUtils.getModuleByName(moduleName, pkgRootDir);
+    var includePaths = options.includePaths.split(path.delimiter); // surprised this is not already an array, coming from broccoli-eyeglass
```

Comment:
consider...

```js
var includePaths = Array.isArray(options.includePaths) ? options.includePaths : options.includePaths.split(path.delimiter);
```
so we're future proofed if down the road it starts coming in as an array rather than a string